pysc2/pysc2/lib/actions.js

```diff
@@ -1,133 +1,134 @@
-const path = require('path');
-const s2clientprotocol = require('s2clientprotocol')
-const Enum = require('python-enum')
+const path = require('path') //eslint-disable-line
+const s2clientprotocol = require('s2clientprotocol') //eslint-disable-line
+const Enum = require('python-enum') //eslint-disable-line
 const point = require(path.resolve(__dirname, './point.js'))
 const pythonUtils = require(path.resolve(__dirname, './pythonUtils.js'))
 const all_collections_generated_classes = require(path.resolve(__dirname, './all_collections_generated_classes.js'))
 const numpy = require(path.resolve(__dirname, './numpy.js'))
 
-const { spatial_pb, ui_pb } = s2clientprotocol
+const { spatial_pb, ui_pb, common_pb } = s2clientprotocol
```

Comment:
Is "common_pb2" instead of "common_pb" ?

Reply:
yup, the javascript build of the protobuf files doesn't use that "2" suffix